### ISSUE TITLE
v0.5 project create not using cli args

### ISSUE THREAD

**Issue by erikerikson** [serverless/serverless#733]
Fetched latest today at 11:21

Executed:
```
slsDev project create -n [name] -d [domain] -s [stage] -r [region] -e [email] -p [profile] -c
```

Expected:
project structure and files created without interaction (since all arguments were provided)

Observed:
```
 _______                             __
|   _   .-----.----.--.--.-----.----|  .-----.-----.-----.
|   |___|  -__|   _|  |  |  -__|   _|  |  -__|__ --|__ --|
|____   |_____|__|  \___/|_____|__| |__|_____|_____|_____|
|   |   |             The Serverless Application Framework
|       |                           serverless.com, v0.5.0
`-------'

Serverless: Initializing Serverless Project...  
Serverless: Enter a unique project bucket name (using a domain is recommended):  ([project]-4yuqaa.com) 
```

**Comment by ac360** [comment]
Thanks @erikerikson You rock :)

**Comment by erikerikson** [comment]
If I then enter a project name, the cli continues but shows an error:
```
Serverless: Enter a unique project bucket name (using a domain is recommended):  ([projectName]-4yuqaa.com) [projectName] 
error:   Bucket name must only contain lowercase letters, numbers, periods and dashes
Serverless: Enter a unique project bucket name (using a domain is recommended):  ([projectName]-4yuqaa.com) 
```

I'll grab debug output

**Comment by erikerikson** [comment]
thanks @ac360

**Comment by erikerikson** [comment]
I would still strongly suggest using https://github.com/jprichardson/node-suppose to automate detection of cli breaking changes.  @ac360 @eahefnawy

**Comment by erikerikson** [comment]
TL;DR:
1. it is unclear why you would ask about creating the default stage and region.
2. why does the short name "4yuqaa" continue showing up?

entering a bucket name was accepted, and it proceeded:
```
Serverless: Enter a unique project bucket name (using a domain is recommended):  ([projectName]-4yuqaa.com) [bucket]
Serverless: Do you want to create a new stage and region for this project? 
  > Yes
    No
Serverless: Creating stage "[stage]"...  
Serverless: Creating region "[region]" in stage "[stage]"...  
Serverless: Notice -- You have chosen not to deploy your resources to CloudFormation.  A CloudFormation template has been saved here: _meta/resources/s-resources-cf-[stage]-[region_without_dashes].json  
Serverless: Successfully created region "[region]" within stage "[stage]"
Serverless: Successfully created stage "[stage]"
Serverless: Successfully initialized project "[projectName]"
```
1. Why did it ask if I wanted to create the thing that used to be required and that I provided information to create?  Am I likely to say, nevermind what I entered?
2. Why did the short name "4yuqaa" keep being used after I supplied a project name?

**Comment by ac360** [comment]
1. It's an easy way to skip creating resources on Infrastructure.  Also, some people may be initializing projects (which uses this same code) that already have stages in their `_meta` folder.
2. This is a bug :(

**Comment by erikerikson** [comment]
1. that makes sense but thought -c was the means of doing that.  except when I'm creating a new project.  maybe context information should be used to decide the behavior of that shared code :)

**Comment by erikerikson** [comment]
I deleted the comment about the follow up error.  I read "project name" where I should have read the actual text of "bucket name".  That error message was appropriate and resolved by using the proper bucket name.

I think it would be appropriate to add a "-b [bucketName]" argument to the project create command.

**Comment by ac360** [comment]
Great suggestions, as usual :)

**Comment by eahefnawy** [comment]
Fixed in fb2d6c5. closing...
